keyboards/matrix/m12og/rev1/mcuconf.h

```diff
@@ -0,0 +1,209 @@
+/*
```

Comment:
This may need to be updated to the latest format

Reply:
right now updating the mcuconf with qmk chibios-confmigrate breaks the build.

Reply:
Yeah the matrix boards with their custom board files aren't to be migrated -- I need to find a better solution.
Ideally their board files evaporate, but I couldn't get binary reproducibility before the last breaking changes merge, without a whole load of duplication. For now, there's no need to upgrade this ` mcuconf.h` -- I'll migrate it with the other matrix boards at a later date, with a proper solution in mind.